jdaviz/configs/cubeviz/plugins/slice/slice.py

```diff
@@ -113,7 +115,9 @@ def _viewer_slices_changed(self, value):
     def _on_select_slice_message(self, msg):
         # NOTE: by setting the slice index, the observer (_on_slider_updated)
         # will sync across all viewers and update the wavelength
-        self.slider = msg.slice
+        with warnings.catch_warnings():
+            warnings.simplefilter('ignore', category=UnitsWarning)
```

Comment:
Maybe this is good enough but if you want to be super specific, you could pass in `message` regex too but I don't think we need to.

Reply:
p.s. I am surprised it is the slider. I thought surely it will be in the parser somewhere. TIL.